domain/ide/command/code_fix_feedback.go

```diff
@@ -0,0 +1,48 @@
+/*
+ * © 2023 Snyk Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package command
+
+import (
+	"context"
+
+	"github.com/snyk/snyk-ls/domain/snyk"
+)
+
+type SnykCodeHttpClient interface {
+	SubmitAutofixFeedback(ctx context.Context, fixId string, positive bool) error
+}
+
+type codeFixFeedback struct {
+	command   snyk.CommandData
+	apiClient SnykCodeHttpClient
+}
+
+func (cmd *codeFixFeedback) Command() snyk.CommandData {
+	return cmd.command
+}
+
+func (cmd *codeFixFeedback) Execute(ctx context.Context) (any, error) {
+	args := cmd.command.Arguments
+	fixId := args[0].(string)
+	positive := args[1].(bool)
+	err := cmd.apiClient.SubmitAutofixFeedback(ctx, fixId, positive)
```

Comment:
If feedback fails, should we really return the error, or just log it and maybe capture it via Sentry?

Reply:
It'll be logged and captured by [the server handler](https://github.com/snyk/snyk-ls/blob/f0ac692403569e2ccb0a72a54a05940977ea7720/application/server/execute_command.go#L53) already.

Reply:
Yes, but should an error be raised to the user? I'd say no.

Reply:
Yes, the user should be aware that we failed to submit the feedback.